src/CliptoExchange.sol

```diff
@@ -43,7 +50,7 @@ contract CliptoExchange {
     /// @notice Register a new creator
     function registerCreator(string memory name, uint256 cost) external {
         // Set a new creator.
-        creators[msg.sender] = Creator({name: name, cost: cost});
```

Comment:
Are we using more than just ETH for payments?

Reply:
I think longer term we'll want to support a future native token. At the moment ETH is primary, we could also consider USDC although that has drawbacks - people might be willing to spend less.